language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -109,6 +109,25 @@ sh_test(
     ],
 ) if not is_windows else None
 
+da_haskell_library(
```

Comment:
:+1: That should avoid the issue. I’m actually wondering if the recent changes in `rules_haskell` to not pass include directories avoids this issue even on Windows. @aherrmann probably knows.